src/Screens/StatusBoard/DoneTabScreen.tsx

```
@@ -0,0 +1,22 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import MainTodoList from '../../Components/Home/MainTodoList';
+import { useTodayListStore } from '../../stores/home';
+
+interface DoneTabScreenProps {}
+
+function DoneTabScreen({}: DoneTabScreenProps) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)